packages/ember-flight-icons/ember-cli-build.js

```diff
@@ -9,7 +9,9 @@ const EmberAddon = require('ember-cli/lib/broccoli/ember-addon');
 
 module.exports = function (defaults) {
   let app = new EmberAddon(defaults, {
-    // Add options here
+    babel: {
+      plugins: [require.resolve('ember-auto-import/babel-plugin')],
```

Comment:
[question] this is useful for local testing of this change within our addons (?), but is there something we need to ask consumer's to do on their side for this change to work properly?

Reply:
I think this is for the dummy app, but the one in the `index.js` file makes it work for all consumers of the addon.

Reply:
And now that I've said that I'm not sure - let me re-read the EAI docs.

Reply:
https://github.com/embroider-build/ember-auto-import#installing-ember-auto-import-in-an-addon - yeah so I can remove this, it's just the `index.js` file change that's necessary.

Reply:
Just pushed an update that removes this.